packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -299,6 +319,54 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
     _addPointer(event);
   }
 
+  /// When multiple pointers down and move, there may be the following situations
+  /// by different platforms:
+  ///   1) They both move:
+  ///   The gesture will move until all the pointers' [PointerMoveEvent] or [PointerPanZoomUpdateEvent] come individually.
+  ///   The gesture will move delta(combined) = max(delta(i) which are positive) + min(delta(i) which are negative),
+  ///   The drag location will be the average of all pointers.
+  ///   Logic:
+  ///     When a [PointerMoveEvent] or [PointerPanZoomUpdateEvent] comes, add the [PointerEvent] in
+  ///     the [_multiPointerMoveTrackers]. When all the pointers' [PointerMoveEvent] come individually,
+  ///     they are considered a batch, the gesture will update by the rules above.
+  ///   Example:
+  ///     When two pointers (p1 and p2) down, the sequence of [PointerMoveEvent]s is : p1 down, p2 down, p1 move +50, p2 move +10,
+  ///     p1 move +30, p2 move -10 , p1 up and p2 up.
+  ///     The [handleEvent] process will be:
+  ///       p1 down ([_multiPointerStartTrackers].add),
+  ///       p2 down([_multiPointerStartTrackers].add),
+  ///       p1 move +50 ([_multiPointerMoveTrackers].add and the gesture wait),
+  ///       p2 move +10 ([_multiPointerMoveTrackers].add, the gesture update +50, and [_multiPointerMoveTrackers].clear),
+  ///       p1 move +30 ([_multiPointerMoveTrackers].add and the gesture wait),
+  ///       p2 move -10 ([_multiPointerMoveTrackers].add, the gesture update +20, and [_multiPointerMoveTrackers].clear),
+  ///       p1 up ([_multiPointerStartTrackers].clear),
+  ///       p2 up.
+  ///
+  ///   2) Only some of pointers moves (it means the pointers' [PointerMoveEvent] or [PointerPanZoomUpdateEvent]
```

Comment:
What do you mean by "the pointers' [PointerMoveEvent] or [PointerPanZoomUpdateEvent]", and why? In what occasion will the "partial batch" occur?